### PR TITLE
[#4779] Improvement(trino-connector): Add the testset for the Iceberg catalog  with jdbc backend

### PR DESCRIPTION
**Pull request by diqiu50**
### What changes were proposed in this pull request?

Add the testset for the Iceberg catalog  with jdbc backend, it reuse the Hive metastore backend testset

### Why are the changes needed?

Fix: #4779 

### Does this PR introduce _any_ user-facing change?

NO

### How was this patch tested?

New It

### CODE REVIEW

File: integration-test/src/test/resources/trino-ci-testset/testsets/lakehouse-iceberg/00000_create_table.sql [hunk 1]
```diff
@@ -1,78 +1,78 @@
-CREATE SCHEMA gt_iceberg.gt_db2;
```
Comment (yuqi1129) [inline integration-test/src/test/resources/trino-ci-testset/testsets/lakehouse-iceberg/00000_create_table.sql:1]: Why do we remove the catalog prefix?

Comment (diqiu50) [reply integration-test/src/test/resources/trino-ci-testset/testsets/lakehouse-iceberg/00000_create_table.sql:1]: I need to reuse this test for the Iceberg catalog with the JDBC back and Hive catalog, So the catalog name should removed

Comment (yuqi1129) [reply integration-test/src/test/resources/trino-ci-testset/testsets/lakehouse-iceberg/00000_create_table.sql:1]: OK

**Review by yuqi1129 — COMMENTED** [review]
LGTM

**Review by yuqi1129 — DISMISSED** [review]
LGTM

**Review by yuqi1129 — APPROVED** [review]
LGTM

**Pull request merged**